iothub_client/src/iothub_client_core_ll.c

```diff
@@ -25,6 +25,8 @@
 #include "internal/iothub_client_diagnostic.h"
 #include "internal/iothubtransport.h"
 
+#include <errno.h>
```

Comment:
Nit: Can be moved along with standard headers at line #7

Reply:
Fixed.